tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
Let me know if this is going to be an issue!

Reply:
I think we should probably chat through the browser support implications at eng sync, but I don't inherently disagree!